tests/ui/test/features/display-no-results.feature

```diff
@@ -0,0 +1,8 @@
+Feature: As a user I want to be able to see no results screen displayed
+
+  Scenario: No results found for non existent organisation search
+    Given I navigate to the organisations page
+    When I add an organisation
+    And I enter "abc" as the organisation name
+    And I submit the search
+    Then a no results found text is displayed on organisation page
```

Comment:
would this read better as Then no results found is displayed on the screen

Reply:
Just wondering if tomorrow we could have another screen with no results found so wanted to be more specific hence why I added 'organisation page'